Neos.ContentRepository.Core/Classes/DimensionSpace/DimensionSpacePoint.php

```diff
@@ -19,8 +19,6 @@
 /**
  * A point in the dimension space with coordinates DimensionName => DimensionValue.
  * E.g.: ["language" => "es", "country" => "ar"]
- *
- * Implements CacheAwareInterface because of Fusion Runtime caching and Routing
```

Comment:
why not?

Reply:
this is just an untrue fact i came across.